### PR TITLE
feat: adds the cart management page

### PR DESCRIPTION
**Pull request by FarhanMobashir**

### CODE REVIEW

File: styles/cart.css [hunk 1]
```diff
@@ -1,3 +1,105 @@
 body {
-  background-color: yellowgreen;
+  position: relative;
+  min-height: 100vh;
+}
+
+#main {
+  padding: 1% 4%;
+  margin: 2rem 0;
```
Comment (nareshpoonia) [inline styles/cart.css:8]: You can consider using an utility class of margin vertical / y axis 
Since it'll be used at many places

**Review by nareshpoonia — APPROVED** [review]
Code looks clean, can use more utilities classes.
Good to merge.

File: styles/cart.css (hunk 1 above)
Comment (FarhanMobashir) [reply styles/cart.css:8]: actually, there are variations at different places and hence can not use a generalized class everywhere

File: pages/cart.html [hunk 2]
```diff
@@ -4,11 +4,262 @@
...
+                <button class="btn btn-text btn-sm">MOVE TO WISHLIST</button>
+                <button class="btn btn-text btn-sm">REMOVE</button>
+              </div>
+            </div>
+          </div>
+          <div class="card-container-horizontal">
+            <img
+              class="card-img card-img-horizontal"
+              src="../assets/jeans-1.png"
+              alt=""
```
Comment (kavyakunder) [inline pages/cart.html:116]: add the description of image in alt

**Review by kavyakunder — APPROVED** [review]
Clean code, looks good to merge.

**Pull request merged**